README.pg_sphere

```diff
@@ -13,7 +13,20 @@ This is an R-Tree implementation using GiST for spherical objects like
 spherical points and spherical circles with useful functions and operators.
 
 NOTICE:
-     This version will work only with PostgreSQL version 9.6 and above.
+     This version will work only with PostgreSQL version 10 and above.
+
+VERSIONING:
+
+Stable versions are marked with tags containing version numbers. Each stable
+version contains upgrade scripts for updating an existing installation to
+a newest version using ALTER EXTENSION UPDATE TO command.
+
+The master branch is intended for development purposes and may contain
+intermediate changes. The current version in the master branch as well as its
```

Comment:
Change "master branch as well as its" to "master branch and its". The "as well as" expression would need commas before and after it, and I think just saying "and" is more succinct.